turnerlabs/yt_push
# DRACO-3605 Add a privacy status field mapping to set the privacy field on uploaded videos.

**Testing instructions**

* Follow the directions in the README to create and map a privacy status field.
* Upload video(s) to YouTube and ensure the value of your privacy status field is reflected in the video uploaded to YouTube.
